modules/eventing/handle_staged.go

```diff
@@ -82,8 +82,9 @@ func (m *Module) processStagedEvent(eventDoc *model.EventDocument) {
 			log.Println("Eventing: Couldn't trigger functions -", err)
 			return
 		}
-
-		result, err := m.functions.CallWithContext(ctxLocal, eventDoc.Service, eventDoc.Function, token, eventDoc)
+		var result interface{}
+		err = m.syncMan.MakeHTTPRequest(ctxLocal, utils.RequestKindDirect, "POST", eventDoc.Url, token, eventDoc, &result)
```

Comment:
Remove the second parameter utils.RequestKindDirect from MakeHTTPRequest function